### ISSUE TITLE
Validation Loss on History Dictionary

### ISSUE THREAD

**Issue by rmozart** [dreamquark-ai/tabnet#234]
## Feature request

Validation Loss Key on the History Dictionary

**What is the expected behavior?**

New key pointing to a np.array with the validation loss stored

**What is motivation or use case for adding/changing the behavior?**

Best way to monitor and evaluate the model then only validation metric

**How should this be implemented in your opinion?**

**Are you willing to work on this yourself?**
yes, however, I would need some assistance

**Comment by Optimox** [comment]
Hi,

I think it's a debatable request.
Each validation/validation metric you add is actually slowing your training process.

Nothing prevents you to add your loss_fn as an evaluation metric, so it seems very easy to me to monitor directly the loss on validation set if you want. I don't see a generic reason to do this systematically.

**Comment by Optimox** [comment]
I'm closing this as I don't see it as high priority, feel free to reopen if you think that adding your loss_fn as a custom evaluation metric does not solve your problem.
